src/charm.py

```diff
@@ -92,7 +94,7 @@ def _on_config_changed(self, event: ops.ConfigChangedEvent):
             self.unit.status = ops.BlockedStatus("invalid log level: '{log_level}'")
 
     @property
-    def _pebble_layer(self):
+    def _pebble_layer(self) -> typing.Union[pebble.LayerDict]:
```

Comment:
```suggestion
    def _pebble_layer(self) -> pebble.LayerDict:
```
What's the point of having Union with one type here?

Reply:
No reason at all... Thanks!